### DISCUSSION TITLE
CALERIE dataset

### DISCUSSION THREAD

**Discussion by RaphaelBecquart** [dayoonkwon/BioAge#12] [General]
Hello,

Thank you so much for having developed this predictor. 

I was wondering whether you could please add the curated CALERIE dataset that you used.

I would be very grateful for this as I am planning to test and perhaps use your code for a publication. 

Kind regards,
Raphael

**Comment by dayoonkwon** [comment]
Hello Raphael,

We cannot share the CALERIE data, but you can apply to access it from the CALERIE bio repository — [CALERIE.duke.edu](http://calerie.duke.edu/)

Best,
Dayoon
